LEAF_Nexus/scripts/refreshOrgchartEmployees.php

```diff
@@ -243,20 +254,33 @@ function updateEmployeeDataBatch(array $localEmployeeUsernames = [])
     if (empty($orgEmployeeDataRes)) {
         return FALSE;
     }
+
+    // regather employee data since we may have inserted items
+    list($localEmpUIDs,$localEmpArray) = getEmployeeUIDs($localEmployeeUsernames);
+
     foreach ($orgEmployeeDataRes as $orgEmployeeData) {
 
-        $localEmployeeDataArray[] = [
-            // this will need to be checked into further I cannot silence this warning without possibly breaking things.
-            'empUID' => $localEmpArray[$orgEmployeeData['userName']],
-            'indicatorID' => $orgEmployeeData['indicatorID'],
-            'data' => $orgEmployeeData['data'],
-            'author' => $orgEmployeeData['author'],
-            'timestamp' => $orgEmployeeData['timestamp'],
-        ];
-    }
+        // if this user is not found, we will skip adding data for them.
+        if(empty($localEmpArray[$orgEmployeeData['userName']]) == true ){
+            continue;
+        }
+        else{
```

Comment:
PSR 12 states that the else needs to be on the same line as the closing { and there should be a space before and after the else.

Reply:
adjusted, I have to review the plugin for this since it used to auto format this for me and did not catch that it did not format.